packages/instantsearch.js/src/connectors/related-products/connectRelatedProducts.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  createDocumentationMessageGenerator,
+  checkRendering,
+  noop,
+} from '../../lib/utils';
+
+import type { Connector, TransformItems, Hit, BaseHit } from '../../types';
+import type {
+  RecommendResultItem,
+  SearchParameters,
+} from 'algoliasearch-helper';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'related-products',
+  connector: true,
+});
+
+export type RelatedProductsRenderState<THit extends BaseHit = BaseHit> = {
+  /**
+   * The matched recommendations from the Algolia API.
+   */
+  recommendations: Array<Hit<THit>>;
```

Comment:
This has been renamed `hits` in `connectFrequentlyBoughtTogether()`.

Reply:
I could not find any piece of info in the RFC favoring either naming. I'm fine with `recommendations` being the exposed prop name. I can switch to this in `frequentlyBoughtTogether`.